### ISSUE TITLE
4.14.0-0.okd-2024-01-06-084517 monitor-plugin failed

### ISSUE THREAD

**Issue by K1kc4** [okd-project/okd#1852]
**Describe the bug**
After updating to 4.14.0-0.okd-2024-01-06-084517 monitor plugin stopped working, from time to time it works for a minute or so, and then goes down again.This happens on 3 of 3 clusters that i updated.

Error: 
```
Failed to get a valid plugin manifest from /api/plugins/monitoring-plugin/
SyntaxError: JSON.parse: unexpected end of data at line 1 column 1 of the JSON data
```
I have checked logs from cluster-monitoring-operator, console-operator and monitor-plugin pods, but there is nothing useful.

**Version**
4.14.0-0.okd-2024-01-06-084517

**How reproducible**
Update to 4.14.0-0.okd-2024-01-06-084517 .
 I did one update from 4.14.0-0.okd-2023-11-14-101924, and two from 4.13.0-0.okd-2023-10-28-065448. 

**Log bundle**
[Download](https://file.io/lYupYNUxXuIq)

**Comment by vrutkovs** [comment]
OCP bug - see https://issues.redhat.com/browse/OCPBUGS-23516

**Comment by titou10titou10** [comment]
Exact same problem here, console says "Dynamic plugin/monitoring plugin degraded" every couple of minutes and the "observe" console menu is on and off...

**Comment by dkoci** [comment]
Same here `Failed to get a valid plugin manifest from /api/plugins/monitoring-plugin/
SyntaxError: The string did not match the expected pattern.`

**Comment by jharmison-redhat** [comment]
Looks like we're just waiting on QA to wrap up getting the fix into the 4.14 stream:
https://github.com/openshift/console/pull/13517

**Comment by stanbog** [comment]
Hello,
@vrutkovs Can you tell me on which 4.14 release this got fixed ?
Right now we've tried:
4.14.0-0.okd-2024-01-06-084517
4.14.0-0.okd-2024-01-26-175629
but both have the issue.

Thank you.
